HackYourFuture-CPH/fp-class20
# Fix Docker for M1 Macs

# Description

If you use Docker to run your DB I found an issue with the version of Mysql being pulled not being compatible with Macs with M1 processer architecture. This PR will ensure a M1 compatible image is used.

I do not know if it will have any adverse effects on other platforms, so it would be nice to have it tested on both intel based macs, PC and linux. If you test it, please specify what platform you tested on.

# How to test?

- Prerequisite: You need to have Docker installed from https://www.docker.com/
- Open command prompt, checkout this branch locally and cd into the server folder `cd packages/server`
- Run `docker-compose up`.
- Verify that you have a running database connected to the app. If you get any errors, let me know.

If you previously have been running Docker for your database you should probably start from scratch:
- `docker-compose down` to make sure your Docker images is shut down.
- `docker system prune -a` to delete all unused images
- `docker volume prune` to delete all unused images
- Now your docker image and volumes should be deleted and you can run `docker-compose up` to start over.

# Checklist
